Source/Python/WarpXWrappers.cpp

```diff
@@ -412,7 +412,7 @@ extern "C"
     }
     void warpx_MoveWindow () {
         WarpX& warpx = WarpX::GetInstance();
-        warpx.MoveWindow (true);
+        warpx.MoveWindow (warpx_getistep(0)+1, true);
```

Comment:
Not too sure about what timestep to put in the python wrapper here.

Reply:
cc @dpgrote

Reply:
The Python interface should mirror the c++ routine, so the step should be passed into warpx_MoveWindow (rather than calling warx_getistep). To do this right, the move_j argument should also be passed in.